source/Engine.cpp

```diff
@@ -239,6 +241,35 @@ namespace {
 
 	const double RADAR_SCALE = .025;
 	const double MAX_FUEL_DISPLAY = 5000.;
+
+	const double CAMERA_VELOCITY_TRACKING = 0.1;
+	const double CAMERA_POSITION_CENTERING = 0.01;
+
+	pair<Point, Point> NewCenter(const Point &oldCenter, const Point &oldCenterVelocity,
+		const Point &baseCenter, const Point &baseVelocity)
+	{
+		if(Preferences::CameraAcceleration() == Preferences::CameraAccel::OFF)
```

Comment:
dont think this should be done here, check before calling the function imo

Reply:
This early return is just a specialization of how the function was originally, it is just a short-circuiting of the normal logic. I would rather not duplicate any chide when it makes no sense to.

Reply:
well it is kind of strange to create a variable newCamera that is only used to transmit data, how about instead of returning a pair you take in newCenter and newVelocity by reference?

Reply:
> well it is kind of strange to create a variable newCamera that is only used to transmit data, how about instead of returning a pair you take in newCenter and newVelocity by reference?

Mutating through references is quite the code smell, and I'd rather just feturn the values and make ut easier for everybody. The code is easier to reason about that way.